test/genesisProtocol.ts

```diff
@@ -170,8 +172,9 @@ describe("GenesisProtocol", () => {
       avatar: dao.avatar.address,
     });
     assert.isOk(result);
-    assert.equal(helpers.fromWei(result.thresholdConstA).toNumber(), 2);
-    assert.equal(result.thresholdConstB, 10);
+    assert.equal(helpers.fromWei(result.thresholdConstA).toNumber(),
```

Comment:
not strictly needed, but consider using https://www.npmjs.com/package/chai-bignumber for asserting on bignumbers.